portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/SharedComponents/PoliciesExpansion.tsx

```diff
@@ -0,0 +1,249 @@
+    const [responseFlowDroppablePolicyList, setResponseFlowDroppablePolicyList] = useState<string[]>([]);
+    const [faultFlowDroppablePolicyList, setFaultFlowDroppablePolicyList] = useState<string[]>([]);
+
+    const { globalLevelPolicies } = useContext<any>(GlobalPolicyContext);
+    const intl = useIntl();
+
+    /**
+     * This is where the applicable (droppable) flows are set for each policy.
+     */
+    useEffect(() => {
+        const requestList = [];
+        const responseList = [];
+        const faultList = [];
+        for (const policy of policyList) {
+            if (policy.applicableFlows.includes('request')) {
+                requestList.push(`policyCard-${policy.id}`);
+            }
+            if (policy.applicableFlows.includes('response')) {
+                responseList.push(`policyCard-${policy.id}`);
+            }
+            if (policy.applicableFlows.includes('fault')) {
+                faultList.push(`policyCard-${policy.id}`);
+            }
+        }
+        setRequestFlowDroppablePolicyList(requestList);
+        setResponseFlowDroppablePolicyList(responseList);
+        setFaultFlowDroppablePolicyList(faultList);
+    }, [policyList]);
+
+    /**
+     * In here, we are populating the attached policy list for each flow.
+     * This will be triggered once we saved a drag`n`droped policy.
+     * This comes after Policies.tsx updateGlobalOperations() method.
+     * This will hold data in the UI until we save the policy.
+     */
+    useEffect(() => {
+        (async () => { 
+            const apiPolicies = globalLevelPolicies;
+
+            /** 
+             * Populate request flow attached policy list.
+             */
+            const requestFlowList: AttachedPolicy[] = [];
+            const requestFlow = apiPolicies.request;
+            for (const requestFlowAttachedPolicy of requestFlow) {
+                const { policyId, policyName, policyVersion, uuid } =
+                    requestFlowAttachedPolicy;
+                if (policyId === null) {
+                    /**
+                     * Handling migration flow.
+                     */
+                    requestFlowList.push({
+                        ...defaultPolicyForMigration,
+                        name: policyName,
+                        displayName: policyName,
+                        applicableFlows: ['request'],
+                        uniqueKey: uuid,
+                    });
+                } else {
+                    const policyObj = allPolicies?.find(
+                        (policy: PolicySpec) => 
+                            policy.name === policyName && 
+                            policy.version === policyVersion,
+                    );
+                    if (policyObj) {
+                        requestFlowList.push({ ...policyObj, uniqueKey: uuid });
+                    } else {
+                        // console.error("Cannot find policyObj for policyId: " + policyId);
```

Comment:
```suggestion
                        
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)